packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-changes-section.tsx

```diff
@@ -103,105 +125,77 @@ const PullRequestChangesSection = ({
   codeOwnerChangeReqEntries,
   codeOwnerPendingEntries,
   codeOwnerApprovalEntries,
-  latestCodeOwnerApprovalArr
-}: PullRequestChangesSectionProps) => {
+  latestCodeOwnerApprovalArr,
+  accordionValues
+}) => {
   // TODO: consider when states change like refetchReviewers
   // refetchCodeOwners
 
-  const getStatusIcon = (status: string) => {
-    switch (status) {
-      // TODO: fix icons to use from nucleo
-      case 'pending':
-        return <Icon name="pending-clock" className="text-icons-alert" />
-      case 'warning':
-        return <Icon name="triangle-warning" className="text-tertiary-background" />
-      case 'error':
-        return <Icon name="triangle-warning" className="text-destructive" />
-      default:
-        return <Icon name="success" className="text-foreground-success" />
-    }
-  }
+  const renderCodeOwnerStatus = useMemo(() => {
```

Comment:
`renderCodeOwnerStatus` sounds like a function, but this is a value. Maybe it would be better to from the `render` part.
```suggestion
  const codeOwnerStatus = useMemo(() => {
```